news/2020-06-19-Bootable-jar-Wildfly-20.adoc

```diff
@@ -0,0 +1,107 @@
+The tooling to build a bootable JAR relies on WildFly Galleon layers to provision the tailored server that is bundled in the JAR file.
+
+== WildFly JAR Maven plugin
+
+The project hosted in link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin[wildfly-extras/wildfly-jar-maven-plugin] defines a new Maven plugin allowing you
+to build a bootable JAR for WildFly (starting version 20.0.0.Final). Although currently incubating, the project has gained enough maturity that you can start using it. Obviously,
+things could still evolve in the near future. It is worth noting that the project has already entered the WildFly 
+feature process in order to become a fully qualified WildFly tooling. The new feature proposal can be found link:https://github.com/wildfly/wildfly-proposals/pull/306[there].
+
+A WildFly bootable JAR contains both the server and your packaged application (a JAR, an EAR or a WAR).
+Once the application has been built and packaged as a bootable JAR, you can start the application using the following command:
+
+```
+java -jar target/myapp-wildfly.jar
+```
+
+A WildFly bootable JAR behave in a way that is similar to a WildFly server installed on file system:
+
+* It supports the main standalone server startup arguments. 
+* It can be administered/monitored using JBoss CLI.
+
+== Getting started
+
+In order to help you start using the ```org.wildfly.plugins:wildfly-jar-maven-plugin``` Maven plugin, we have defined a 
+set of link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/tree/2.0.0.Alpha4/examples[examples] that cover common use-cases. 
+
+To retrieve the examples:
+
+```
+git clone -b 2.0.0.Alpha4 http://github.com/wildfly-extras/wildfly-jar-maven-plugin
+cd wildfly-jar-maven-plugin/examples
+```
+
+A good example to start with is the link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/tree/2.0.0.Alpha4/examples/jaxrs[jaxrs] example.
+To build and run the jaxrs example:
+
+```
+cd jaxrs
+mvn package
+java -jar target/jaxrs-wildfly.jar
+```
+
+The plugin documentation (an ```index.html``` file to download) can be found link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/releases/download/2.0.0.Alpha4/index.html[there].
+It contains a comprehensive list of the options you can use to fine tune the Maven build and created bootable JAR.
+
+Be sure to read the link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/tree/2.0.0.Alpha4/examples/README.md[examples/README] 
+that contains required information to run in an OpenShift context. 
+
+== Main features
+
+The main features are covered in details in the plugin link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/releases/download/2.0.0.Alpha4/index.html[documentation]. 
+Here is a summary of what the Maven plugin and created bootable JAR offer.
+
+=== Maven Plugin
+
+* Supports WildFly Galleon layers to create use-case tailored bootable JAR.
```

Comment:
I am not sure but maybe it actually `Leverages Wildfly Galleon layers to...`. WDYT?